liveagent-plugin-versions.user.js

```diff
@@ -393,7 +394,10 @@
 
     /**
      * === Changelog ===
-     *
+     * 
+     * 2.22.2.0 - 2019-10-06
```

Comment:
Third digit is for hotfixes or feature releases.
This release was a maintenance release, so the second digit should be increased.